src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/PaymentsTest.java

```diff
@@ -0,0 +1,152 @@
+package uk.gov.hmcts.reform.bulkscan.endtoendtests;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import com.warrenstrange.googleauth.GoogleAuthenticator;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.Await;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.Container;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.StorageHelper;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.ZipFileHelper;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.utils.ProcessorEnvelopeResult;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.format;
+import static org.apache.http.HttpHeaders.CONTENT_TYPE;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.apache.http.entity.ContentType.APPLICATION_FORM_URLENCODED;
+import static org.apache.http.entity.ContentType.APPLICATION_JSON;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.bulkscan.endtoendtests.utils.ProcessorEnvelopeStatusChecker.getZipFileStatus;
+
+public class PaymentsTest {
+
+
+    private static final String IDAM_API_URL = "https://idam-api.aat.platform.hmcts.net";
+    private static final String S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal";
+    private static final String CORE_CASE_DATA_API_URL = "http://ccd-data-store-api-aat.service.core-compute-aat.internal";
+    private static final String REDIRECT_URI = "http://localhost/receiver";
+    public static final String BEARER = "Bearer ";
+
+
+    private Config conf = ConfigFactory.load();
```

Comment:
we normally move those urls to config. and if we do then there's a use of that `ConfigFactory.load()` here